src/shared/components/app/app.tsx

```diff
@@ -33,12 +33,14 @@ export class App extends Component<any, any> {
       <>
         <Provider i18next={I18NextService.i18n}>
           <div id="app" className="lemmy-site">
-            <a
-              className="skip-link bg-light text-dark p-2 text-decoration-none position-absolute start-0 z-3"
+            <button
+              type="button"
+              tabIndex="0"
```

Comment:
Don't buttons have tabIndex 0 by default?

Reply:
Ah of course, I was using this to debug some weird behavior but can remove it now, thanks.